examples/tools/android_native_run/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+find_package (JNI REQUIRED)
+find_library (LIBMYFUNCTIONS_LIBRARIES libmyfunctions.so)
+
+add_executable(android_native_run
+ main.cpp
+ classpath.cpp
+ options.cpp
+ JVM.cpp
+ Other_libraries.cpp
+ )
+
+ add_library(myfunctions SHARED myfunctions.cpp)
+
+
+configure_file(path.h.in ${CMAKE_CURRENT_BINARY_DIR}/path.h @ONLY)
+
+
+target_link_libraries(android_native_run PRIVATE QBDL dl ${JNI_LIBRARIES})
+target_link_libraries(android_native_run PRIVATE myfunctions)
```

This can be merged into a single statement:
```suggestion
target_link_libraries(android_native_run PRIVATE QBDL dl myfunctions ${JNI_LIBRARIES})
```
(obviously you'll have to change the `myfunctions` name if you address my previous comment :)

---

Done